typescript/cli/src/commands/registry.ts

```diff
@@ -71,19 +72,34 @@ const listCommand: CommandModuleWithContext<{ type: ChainType }> = {
 /**
  * Addresses command
  */
-const addressesCommand: CommandModuleWithContext<{ name: string }> = {
+const addressesCommand: CommandModuleWithContext<{
+  name: string;
+  contract: string;
+}> = {
   command: 'addresses',
+  aliases: ['address', 'addy'],
   describe: 'Display the addresses of core Hyperlane contracts',
   builder: {
     name: {
       type: 'string',
       description: 'Chain to display addresses for',
       alias: 'chain',
     },
+    contract: {
+      type: 'string',
+      description: 'Specific contract to print addresses for',
+      demandOption: false,
+    },
   },
-  handler: async ({ name, context }) => {
+  handler: async ({ name, context, contract }) => {
     if (name) {
       const result = await context.registry.getChainAddresses(name);
+      if (contract && result?.[contract]) {
```

Comment:
Seems like a global setting for machine readability would be useful but this is fine for now if it solves a problem :)

Reply:
agreed but will punt for now
I didnt remove the top-level log